build.gradle

```diff
@@ -391,6 +391,9 @@ test {
         'src/test/xsl/common/dita-utilities.xspec',
         'src/test/xsl/common/uri-utils.xspec',
         'src/test/xsl/plugins/org.dita.html5/xsl/functions.xspec',
+        'src/test/xsl/plugins/org.dita.html5/xsl/simpletable.xspec',
+        'src/test/xsl/plugins/org.dita.pdf2/xsl/fo/simpletable.xspec',
```

Comment:
these two were already there but they were not included in this file -- which I assume determines which tests to run